python/openassetio/test/manager/_implementation.py

```diff
@@ -22,10 +22,10 @@
 
 from openassetio import hostAPI, pluginSystem, logging
 
-from ..specifications import TestHarnessLocale
+from .specifications import ManagerTestHarnessLocale
```

Comment:
@feltech I meant to ask before, I feel we need an official stance on relative vs absolute imports

Reply:
I don't mind the Python 3 explicit relative imports, but really I also don't mind absolute imports.  

I think on balance relative imports convey slightly more information at a glance (where the file you're looking at sits in relation to its dependencies), and is terser. So I guess I lean toward using them.

One minor disadvantage is the "attempted relative import with no known parent package" error if importing the module on its own. Perhaps this could be annoying if trying to debug something on the command line, but is easily avoided. I've had this erroneously flagged by `pylint` before, but am struggling to reproduce it now.

Reply:
Cool, shall we say relative by default then? I guess it also means you dont bake in the outer package name into every file, and force it's `__init__`.

Reply:
Agreed